code/modules/martial_arts/martial.dm

```diff
@@ -264,6 +275,7 @@
 
 	var/datum/martial_art/the_sleeping_carp/theSleepingCarp = new(null)
 	theSleepingCarp.teach(user)
+	user.faction += list("carp") //makes space carp peaceful to sleeping carp users
```

Comment:
Why's this a list? If you're wanting to add that to the user's factions rather than replacing it then I'd recommend something like this instead:
```suggestion
	user.faction |= "carp" //makes space carp peaceful to sleeping carp users
```
*(The [`|=` operator](http://www.byond.com/docs/ref/#/operator/|=) adds `"carp"` to the factions if it's not already there.)*